tests/functional/test_rose_stem.py

```diff
@@ -251,234 +252,83 @@ def _inner_fn(rose_stem_opts, verbosity=verbosity):
     yield _inner_fn
 
 
-@pytest.fixture(scope='class')
-def rose_stem_run_really_basic(rose_stem_run_template, setup_stem_repo):
+def test_rose_stem_run_really_basic(rose_stem_run_template, setup_stem_repo):
+    """Check that assorted variables have been exported."""
     rose_stem_opts = {
         'stem_groups': [],
         'stem_sources': [
             str(setup_stem_repo['workingcopy']), "fcm:foo.x_tr@head"
         ],
     }
-    yield rose_stem_run_template(rose_stem_opts)
+    plugin_result = rose_stem_run_template(rose_stem_opts)
+    assert plugin_result['run_stem'].returncode == 0
 
 
-class TestReallyBasic():
-    def test_really_basic(self, rose_stem_run_really_basic):
-        """Check that assorted variables have been exported.
-        """
-        assert rose_stem_run_really_basic['run_stem'].returncode == 0
-
-
-@pytest.fixture(scope='class')
 def rose_stem_run_basic(rose_stem_run_template, setup_stem_repo):
+    """Check that assorted variables have been exported."""
     rose_stem_opts = {
         'stem_groups': ['earl_grey', 'milk,sugar', 'spoon,cup,milk'],
         'stem_sources': [
             str(setup_stem_repo['workingcopy']), "fcm:foo.x_tr@head"
         ],
         'workflow_name': setup_stem_repo['suitename']
     }
-    yield rose_stem_run_template(rose_stem_opts)
-
-
-class TestBasic():
-    @pytest.mark.parametrize(
-        'expected',
-        [
-            "run_ok",
-            "RUN_NAMES=['earl_grey', 'milk', 'sugar', 'spoon', 'cup', 'milk']",
-            "SOURCE_FOO=\"{workingcopy} fcm:foo.x_tr@head\"",
-            "HOST_SOURCE_FOO=\"{hostname}:{workingcopy} fcm:foo.x_tr@head\"",
-            "SOURCE_FOO_BASE=\"{workingcopy}\"\n",
-            "SOURCE_FOO_BASE=\"{hostname}:{workingcopy}\"\n",
-            "SOURCE_FOO_REV=\"\"\n",
-            "SOURCE_FOO_MIRROR=\"fcm:foo.xm/trunk@1\"\n",
-        ]
```

Comment:
This parameterisation only defines on variable, `expected`, which is always `run_ok`.

I don't think the other stuff did anything?

By removing this, have I killed what this was supposed to be testing?

Reply:
Yes, I had misread that parametrisation, drafted.